test/src/main/resources/StageSpecifics_LOC.properties

```diff
@@ -45,15 +45,28 @@ scenariosroot3.m2e.pom.properties=../META-INF/maven/org.ibissource/ibis-adapterf
 scenariosroot4.directory=../../../../src/test/resources/TestTool
 scenariosroot4.description=Jetty Maven Plugin
 
-local.temp=c:/temp
-#local.temp=C:/Data/BuildCompare/temp
-#scenariosroot2.directory=${local.temp}/testtool
-#scenariosroot2.description=no-D-partition
-
 #larva.adapter.active=false
 #larva.adapter.scheduler.active=false
 larva.adapter.mail.active=false
 larva.adapter.writeToSystemOut=true
 larva.adapter.execute=IbisConsole
 # Other examples:
 #larva.adapter.execute=EsbSoapValidator
+
+testdata.dir.match=${log.dir.match}[\\\\/]testdata
+
+# catalina.properties you need for Windows:
+# -----------------------------------------
+
+# otap.stage=LOC
+# log.dir=c:/temp
+#
+# The directory represented by ${log.dir} can appear in different ways under Windows.
+# Supply this regex to match every possible way.
+#
+# Comment on the value chosen for Windows. This expression matches:
+#   c:/temp
+#   c:\temp
+#   C:/temp
+#   C:\temp
+# log.dir.match=[cC]:[\\\\/]temp
```

Comment:
Je gebruikt hier nu een regex maar is dat nog wel nodig? Geven verschillende adapters de resultaten in een ander format terug? (hoofd/kleine-letters en back/forward-spaces)